### ISSUE TITLE
Change scaladoc link title to "API Documentation"

### ISSUE THREAD

**Issue by arosien** [typelevel/cats#2004]
https://typelevel.org/cats/ has a link to the scaladoc, but the link text "Documentation" is confusing. Most projects refer to the scaladoc as "API Documentation".

**Comment by jiminhsieh** [comment]
@arosien Do you mean the "Documentation" link at the right side of GitHub linked to https://typelevel.org/cats/api/cats/index.html in header?

**Comment by arosien** [comment]
@jiminhsieh, yes, that link. My issue is that the page itself (https://typelevel.org/cats/) *is* the documentation, and that link is to the scaladoc / API documentation, so I want the link to be more precise.

**Comment by Obarros** [comment]
I can take a look on this one.

But maybe it will require a change on the sbt-microsite plugin.

https://47deg.github.io/sbt-microsites/docs/settings.html

**Comment by LukaJCB** [comment]
Yes, I think we should add an issue to microsites to figure out if we can add a feature for it :)

**Comment by Obarros** [comment]
Great, I create the issue.

https://github.com/47deg/sbt-microsites/issues/267

I will be happy to create the PR there, let's see the feedback and update here later.

**Comment by Obarros** [comment]
@LukaJCB, new version of the `sbt-microsites plugin` (0.7.20) with the PR merged released with the change we need to make this happen! 👍 

I will create a PR to address it!
